src/test/java/com/appdynamics/monitors/azure/AzureMonitorTest.java

```diff
@@ -1,37 +1,33 @@
+/*
+ * Copyright 2018. AppDynamics LLC and its affiliates.
+ *  All Rights Reserved.
+ *  This is unpublished proprietary source code of AppDynamics LLC and its affiliates.
+ *  The copyright notice above does not evidence any actual or intended publication of such source code.
+ *
+ */
+
 package com.appdynamics.monitors.azure;
 
```

Comment:
There should be comprehensive unit tests for your AzureMetricsProcessor, as that's where most of the heavy lifting is happening. You'll probably have to mock a connection to the Azure API and create some dummy data in your test resources. Again, the Netscaler extension is a good reference point.